aws-core/src/main/java/com/amplifyframework/auth/AWSCredentials.kt

```diff
@@ -85,3 +86,12 @@ class AWSTemporaryCredentials(
      */
     val expiration: Instant
 ) : AWSCredentials(accessKeyId, secretAccessKey)
+
+internal fun AWSCredentials.toCredentials(): Credentials {
```

Comment:
`Credentials` as a return type makes sense b/c it's from the Kotlin SDK. For the method name, I might consider something a more specific, especially since it's internal: `toSdkCredentials` or similar?

Reply:
Sure, this is pre-existing and moved files where I felt like it belongs, but I like the suggestion.